homework1/src/app/play-again/play-again.component.ts

```diff
@@ -0,0 +1,36 @@
+import { ScaleControllerService } from "./../scale-controller.service";
+import { GameStateService } from "./../game-state.service";
+import { GameState } from "../game-state";
+import { Component, OnInit, Input, Output } from "@angular/core";
+import { EventEmitter } from "@angular/core";
+
+@Component({
+  selector: "app-play-again",
+  templateUrl: "./play-again.component.html",
+  styleUrls: ["./play-again.component.scss"],
+})
+export class PlayAgainComponent implements OnInit {
```

Comment:
do not create a component just for a button, you can move its logic to clicker component

Reply:
done